packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -131,8 +131,8 @@ export class ColumnHeaderCell extends React.Component<IColumnHeaderCellProps, IC
     };
 
     public render() {
-        const { isActive, isColumnSelected, loading, resizeHandle, style } = this.props;
-        const classes = classNames(Classes.TABLE_HEADER, {
+        const { className, isActive, isColumnSelected, loading, resizeHandle, style } = this.props;
+        const classes = classNames(className, Classes.TABLE_HEADER, {
```

Comment:
our convention is to always put `this.props.className` last so it can override any conditional classes.

if, for instance, i tried to set `className={Classes.TABLE_HEADER_ACTIVE}` here but line 136 resolved to `false` then that class would _not_ be set on the element.

but if `className` is last then it overrides all previous conditionals.